nsv13/code/game/objects/items/nsv13_plushes.dm

```diff
@@ -0,0 +1,9 @@
+/obj/item/toy/plush/random
```

Comment:
This does not need an entire file just for itself.
Use nsv13/code/game/objectives/items/custom_items.dm